feat(v1): allow specifying meta desc in front matter (#1859)
facebook/docusaurus: docs/api-doc-markdown.md

`title`: The title of your document. If this field is not present, the document's `title` will default to its `id`.

> `description`: The description of your document which will become the `<meta name="description" content="..."/>` and `<meta property="og:description" content="..."/>` in `<head>`. If this field is not present, it will default to the first line of the contents.

Reviewer: Maybe should make it similar to v2

https://github.com/facebook/docusaurus/pull/1862/files

> `description`: The description of your document which will become the `<meta name="description" content="..."/>` and `<meta property="og:description" content="..."/>` in `<head>`. If this field is not present, it will default to the first line of the contents.

Reviewer: ```suggestion
`description`: The description of your document which will become the description meta tag. If this field is not present, it will default to the first line of the contents.
```

Author: I prefer to keep it explicit
